src/meta/geo/forest.ts

```diff
@@ -1,3 +1,19 @@
+export interface ForestOptions {
+  sources: ForestSources[]
+}
+
+export type ForestSources =
+  | null
+  | 'JAXA'
+  | 'TandemX'
+  | 'GlobeLand'
+  | 'ESAGlobCover'
+  | 'Copernicus'
+  | 'ESRI'
+  | 'ESAWorldCover'
+  | 'Hansen'
+  | 'MODIS'
+
```

Comment:
Maybe is possible to reuse the ForestSource enum values?

Reply:
Yes, I haven't worked with enums much, but it seems to be interchangeable with type for this purpose, so I've done it.